index.js

```diff
@@ -129,9 +129,17 @@ function encodeName (name) {
 }
 
 function prebuildName (target, opts) {
-  var tags = [encodeName(opts.name || target.runtime)]
+  var tags = []
 
-  if (!opts.napi) {
+  if (opts.name) {
+    tags.push(encodeName(opts.name))
+  }
+
+  tags.push(target.runtime)
```

Comment:
if named it shouldnt tag with the runtime (soft deprecated anyway as electron is node compat now)

Reply:
I can remove that part.

Though I just want to clarify, dropping the runtime will cause `prebuild/node-gyp-build` to no longer detect the `runtime` and drops the specificity, this is the expectation?

Reply:
I've re-added the logic for encoding `runtime` only if `name` is not provided. Though it will seemingly always have a name unless the `package.json` is missing or not correctly formatted.